CMSgov/mdct-mcr
# redline section A

## Description
Updated per redline spec for section A

### How to test
Go to 'Point of Contact' and 'Add BSS Entities' sections, see that they both have the spreadsheet widget w/ new text and formatting.


## Code author checklist

## Reviewer checklist (two different people)
